service-base/src/test/java/org/eclipse/hono/service/registration/AbstractRegistrationServiceTest.java

```diff
@@ -208,7 +210,10 @@ public void testGetSucceedsForRegisteredDeviceWithData(final VertxTestContext ct
 
                     assertNotNull(s.getPayload());
                     assertEquals(vias, s.getPayload().getVia());
+                    assertEquals(viaGroups, s.getPayload().getViaGroups());
 
+                    ctx.completeNow();
+                    /*
                     getRegistrationService().assertRegistration(TENANT, deviceId, ctx.succeeding(s2 -> {
```

Comment:
I guess the "getRegistrationService().assertRegistration" block can be activated again (and the above `ctx.completeNow();` removed) ?